packages/docusaurus-theme-classic/src/theme-classic.d.ts

```diff
@@ -1072,6 +1072,7 @@ declare module '@theme/NavbarItem/LocaleDropdownNavbarItem' {
   export interface Props extends DropdownNavbarItemProps {
     readonly dropdownItemsBefore: LinkLikeNavbarItemProps[];
     readonly dropdownItemsAfter: LinkLikeNavbarItemProps[];
+    readonly queryString: string;
```

Comment:
```suggestion
    readonly queryString?: string;
```

The props can be undefined here because the Joi schema does not use '' as the default value